ambry-clustermap/src/main/java/com/github/ambry/clustermap/MySqlReportAggregatorTask.java

```diff
@@ -87,6 +87,7 @@ public TaskResult run() {
         statsWrappers.put(instanceName,
             accountStatsStore.queryStatsOf(clusterMapConfig.clusterMapClusterName, instanceName));
       }
+      logger.info(String.format("Aggregating stats from %d hosts", statsWrappers.size()));
```

Comment:
why not ```logger.info("Aggregating stats from {} hosts", statsWrappers.size());```

Reply:
does it matter? is one better than the other?

Reply:
String.format is generally redundant inside logger calls.